### ISSUE TITLE
Resolver: Resolver on display 2 iconifies whenever anything on Display 1 is touched

### ISSUE THREAD

**Issue by gitvanb** [icpctools/icpctools#308]
On windows, I am trying to run the resolver on Display 2 while doing other things on Display 1. As soon as I give focus to anything on D1, the Resolver iconifies. If I'm trying to share it, that is a very bad look.

**Comment by deboer-tim** [comment]
AFAIK there's nothing in the code to cause this nor any normal reason it should happen. Can't reproduce on my Mac and I don't have any dual-display Windows machines at the moment, will try to figure out how to test.

**Comment by deboer-tim** [comment]
I set this up temporarily, and confirmed that this is actually 'normal'/expected behaviour on Windows. I found several apps that have the same problem and Stack Overflow posts that confirm it.

The problem is that Windows automatically minimizes a full screen exclusive window when you try to access another program, which maybe makes some sense when you only have one display, but absolutely none when you have two or more.

The workaround that some other programs have implemented is to not use full screen exclusive mode when you have multiple monitors and just use a maximized window. The fix is in build 2.3.446.
